package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "2.6.18",
+  "version": "2.6.19",
```

Comment:
Sorry, already released .19 cause we were blocking parsing. It's a big one, let's give it a minor version bump?

Reply:
Changed it 2.7.0